booktrader/src/pages/Home/index.js

```diff
@@ -1,7 +1,59 @@
 import React from "react";
 import NavBar from "../../Component/Navbar";
+import { Jumbotron, Container } from "reactstrap";
+import cover from "../../assets/cover.png";
+import Carousel from "../../Component/Slideshow";
+import Jumbo from "../../Component/Jumbo";
+import Footer from "../../Component/Footer";
+const list_item = [
+  {
+    link: "/store",
+    name: "Store"
+  },
+  {
+    link: "/login",
+    name: "Log in"
+  },
+  {
+    link: "/signup",
+    name: "Sign up"
+  }
+];
 const Home = () => {
-  return <NavBar item1="Store" item2="Log in" item3="Sign up" />;
+  return (
+    <React.Fragment>
+      <Jumbotron
+        style={{
+          background: `url(${cover}) center center / cover no-repeat`,
+          height: "650px"
+        }}
+        fluid
+      >
+        <NavBar
```

Comment:
Why you do not parse all list_item to NavBar component and after that, you just you use map for this